src/components/Avatar/tests/Avatar.test.tsx

```diff
@@ -1,28 +1,25 @@
 import React from 'react';
-// eslint-disable-next-line no-restricted-imports
-import {mountWithAppProvider} from 'test-utilities/legacy';
 import {mountWithApp} from 'test-utilities';
 import {Avatar, Image} from 'components';
 
 describe('<Avatar />', () => {
   describe('intials', () => {
     it('renders intials if the image is not provided', () => {
-      const avatar = mountWithAppProvider(<Avatar initials="DL" />);
-      expect(avatar.find('span[role="img"] span svg')).toHaveLength(1);
+      const avatar = mountWithApp(<Avatar initials="DL" />);
+      expect(avatar).toContainReactComponent('text', {children: 'DL'});
```

Comment:
This tests the implementation details of `Avatar` pretty closely. Maybe a `toContainReactText` would be better suited for what you're testing here?